azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -1851,6 +1992,44 @@ func flattenBlobPropertiesDeleteRetentionPolicy(input *storage.DeleteRetentionPo
 	return deleteRetentionPolicy
 }
 
+func flattenBlobPropertiesChangeFeed(input *storage.ChangeFeed) []interface{} {
+	changeFeed := make([]interface{}, 0)
+
+	if input == nil || input.Enabled == nil || !*input.Enabled {
+		return changeFeed
+	}
+
+	days := 0
+	if input.RetentionInDays != nil {
+		days = int(*input.RetentionInDays)
+	}
+
+	return []interface{}{map[string]interface{}{
+		"retention_in_days": days,
+	},
```

Comment:
_minor_ but I'm surprised the linter isn't catching this:

```suggestion
	return []interface{}{
		map[string]interface{}{
			"retention_in_days": days,
		},
```